src/components/Auth/Login/Login.react.js

```diff
@@ -317,7 +317,7 @@ class Login extends Component {
                     : '#19314B'
                 }
                 labelColor="#fff"
-                disabled={!isValid || loading}
+                disabled={!isValid || loading || this.state.password.length < 6}
```

Comment:
Don't put this here. Take reference from the code of Signup and see how we have handled this case.

Reply:
@AakashMallik  Okay Sure.